packages/app/src/renderer/wrappers/root-wrapper.tsx

```diff
@@ -145,6 +148,18 @@ const RootWrapper: FunctionComponent<Props> = ({
     connect()
   }, [])
 
+  useEffect(() => {
+    let interval: NodeJS.Timeout
+
+    if (connectedAndUnlocked) {
+      interval = setInterval(() => updateAllIndexes(), 60000)
```

Comment:
Maybe we can decrease this time to 30s ? 🤔

Reply:
I suggest to check how an application will be work and then makes some improvements